src/custom/hooks/useTransactionConfirmationModal.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useCallback } from 'react'
```

Comment:
simply extrapolated out of swapmod where it was being used for approvals.

it hasnt been used in `SwapMod` - only in Claiming - as i figured it's out of scope of this PR (this kinda is too but hey whatever it's way nicer this way)